src/style.css

```
@@ -1,3 +1,80 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --header-height: 60px;
+  --link-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  text-transform: uppercase;
+}
+
+h1 {
+  text-align: center;
+}
+
+.body {
+  margin: 0;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.moyo {
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+}
+
+.nav_item {
+  cursor: pointer;
+}
+
+.nav_item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav_link {
+  height: var(--header-height);
+  line-height: var(--header-height);
+  font-size: 12px;
+  text-decoration: none;
+  color: #000;
+  display: inline-block;
+  position: relative;
+}
+
+.nav_link.is-active {
```

```suggestion
.nav_link.is-active,
.nav_link:hover {
```
You can group your selectors with commas, to avoid repeating code